qcfractal/storage_sockets/sql_models.py

```diff
@@ -737,6 +776,18 @@ def base_result(self, val):
     base_result_id = Column(Integer, ForeignKey("base_result.id"), unique=True)
     base_result_obj = relationship(BaseResultORM, lazy='select')  # or lazy='joined'
 
+
+    # An important special case is ORDER BY in combination with LIMIT n: an
+    # explicit sort will have to process all the data to identify the first n
+    # rows, but if there is an index matching the ORDER BY, the first n rows
+    # can be retrieved directly, without scanning the remainder at all.
+
+    __table_args__ = (
+        Index('ix_task_queue_created_on', "created_on"),
+        Index('ix_task_queue_status', "status"),
+        Index('ix_task_queue_manager', "manager"),
```

Comment:
We look up based on tag, program, procedure, status, priority, and base_result_id. Do we need additional indices here?

Reply:
Is this a multi index? We look up by `status` and `base_result_id` separately.

The main multi index is [here](https://github.com/MolSSI/QCFractal/blob/master/qcfractal/storage_sockets/sqlalchemy_socket.py#L1516)

Reply:
`Status` will work from the multi-column index because it's the left most one. I forgot to add the `base_result_id`, I'll add it.